specs/regional_communities.md

```diff
@@ -1,61 +1,333 @@
+- If splitting along the Constellations dimension then
+   - if there is already a constellation picked, then simply ask what constellation the new Community should serve and it becomes a sibling Community. Example: [Near Somalia DeFi] splits into [Near Somalia DeFi] + [Near Somalie Gaming]
+   - if there was no prior constellation picked, then ask which Constellations the new Community should serve and becomes a child Community. Example: [Near Uganda] splits into [Near Uganda] + [Near Uganda NFT]
+- The split has to be along at one and only one of the dimensions. The split cannot proceed unless there is an actual change proposed, one that will make the new Community different by design from the original Community from which it splits.
+
+Any member can request a split, it must not be approved by the admins. 
+- The members voting should be asked a few questions
+   - Do you approve of this split proposal?
+   - If so, would you want to become a member of the new Community? Would you voluenteer to also become and admin?
+   - If so, would you also want to stay a member of the current community?
+- It is this author's opinion that a minimum of three to five other members should approve the split and also wish to become a member of the new Community for it to take effect
+- Once sufficiently approved, then the split should immediately be executed. It should not require admin involvement or approval, because we want the split to be a natural, organic and permissionless occurence.
+
+The user requesting the split trigger an automatic vote to approve the split. If the request is approved then the original requestor will also be the first leader of the new community by default. 
+ 
+After splitting the community, then each member should be given a choice as to which of the resulting communities they wish to belong to, or if they wish to be a member of both.
 
-List all users and their types
+### Merge two Communities
+We don't anticipate this to happen often, but it could be the case that two communities have shrinking membership and would like to merge to increase the total membership in one Community.
 
-### Actions
+A merge proposal should be approved by majority vote in both communities before it is allowed to happen.
 
-List the actions each collective will take individually.
+It's not easy to design rules for how the merge should be executed, based on the complexity of possible permutations of the pre-merge Communities. It seems plausible that merge power should not be given to regular users but could be given as an admin power to a dev / superuser. The alternative would be that merging not be supported, in which case the community will manually have to move from and abandon one community in favor of the other.
```

Comment:
I think merging should be a "referendum" action

Reply:
Yeah. As in advisory, non-binding.